smallboard/urls.py

```diff
@@ -19,15 +19,16 @@
 from django.urls import path, include
 
 from . import views
+from hunts.views import ActiveHuntRedirectView
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
-    path('accounts/', include('django.contrib.auth.urls')),
-    path('accounts/', include('accounts.urls')),
-    path('puzzles/', include('puzzles.urls')),
-    path('hunts/', include('hunts.urls')),
-    path('answers/', include('answers.urls')),
-    path('privacy', views.privacy, name='privacy'),
-    path('', lambda r: HttpResponseRedirect('hunts/{}/'.format(settings.ACTIVE_HUNT_ID))),
-    path('', include('social_django.urls', namespace='social')),
+    path("admin/", admin.site.urls),
+    path("accounts/", include("django.contrib.auth.urls")),
+    path("accounts/", include("accounts.urls")),
+    path("puzzles/", include("puzzles.urls")),
+    path("hunts/", include("hunts.urls", namespace="hunts")),
+    path("answers/", include("answers.urls")),
```

Comment:
Is the namespace="hunts" part necessary? Or else, should we also add namespace="answers" for consistency?